app/services/sync/fetch_profile.rb

```diff
@@ -8,11 +8,12 @@ def call
       self.profile = Profile.find_by(nickname: profile_name)
       self.github_profile = Github::ProfileConsumer.new(profile_name).call
 
-      return if profile.present? && github_profile.nil?
+      return if profile.nil? && github_profile.nil?
```

Comment:
IMO it's better to stick to `blank?`
```suggestion
      return if profile.blank? && github_profile.blank?
```

Reply:
oh! Done